modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -312,6 +315,26 @@ func (h *Handler) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 				}
 				h.HealthChecks.Active.Port = portNum
 
+			case "health_headers":
+				healthHeaders := make(http.Header)
+				for d.Next() {
+					for d.NextBlock(0) {
+						key := d.Val()
+						values := d.RemainingArgs()
+						if len(values) == 0 {
+							values = append(values, "")
+						}
```

Comment:
Is this necessary? Did you try without this to see if Go will still set the header with an empty value?

Reply:
if I remove this logic, the empty header value will be `null` in json format caddyfile
![image](https://user-images.githubusercontent.com/2509813/103434156-4d147d00-4c38-11eb-9501-4364989bf348.png)
![image](https://user-images.githubusercontent.com/2509813/103434148-1fc7cf00-4c38-11eb-92a7-bad60785eb3d.png)

Reply:
Interesting... ok. thanks! I keep forgetting how that works.